ocs_ci/ocs/constants.py

```diff
@@ -193,6 +193,8 @@
 INFRA_MACHINE = "infra"
 MOUNT_POINT = "/var/lib/www/html"
 TOLERATION_KEY = "node.ocs.openshift.io/storage"
+CL_SUB = "cluster-logging"
+ES_SUB = "elasticsearch-operator"
```

Comment:
the var names are not clear and can be difficult to understand, please give clear names which can be easily understandable.